File: ce/outlook-app/snippets/csharp/CRMV8/query/cs/linqexamples34.cs
// <snippetlinqexamples34>


 using (OrganizationServiceContext orgSvcContext = new OrganizationServiceContext(_serviceProxy))
 {

  var list_getattrib1 = (from c in orgSvcContext.CreateQuery("contact")
                         where c.GetAttributeValue<Guid?>("contactid") != _contactId1
                         select new { 
                          FirstName = c.GetAttributeValue<string>("firstname"), 
                          LastName = c.GetAttributeValue<string>("lastname") 
                         }).ToList();
  foreach (var c in list_getattrib1)
  {
   System.Console.WriteLine(c.FirstName + " " + c.LastName);
  }
 }

// </snippetlinqexamples34>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/linqexamples34.vb
' <snippetlinqexamples34>


 Using orgSvcContext As New OrganizationServiceContext(_serviceProxy)
  Dim list_getattrib1 = ( _
      From c In orgSvcContext.CreateQuery("contact") _
      Where Not c.GetAttributeValue(Of Guid?)("contactid") _
      .Value.Equals(_contactId1) _
      Select New With
             {
              Key .FirstName = c.GetAttributeValue(Of String)("firstname"),
              Key .LastName = c.GetAttributeValue(Of String)("lastname")}
          ).ToList()
  For Each c In list_getattrib1
   Console.WriteLine(c.FirstName &amp; " " &amp; c.LastName)
  Next c
 End Using

' </snippetlinqexamples34>